swan-cern/Dockerfile

```diff
@@ -53,14 +53,20 @@ ENV DASK_DIR=/srv/dask
 ENV DASK_LIB_DIR=${DASK_DIR}/lib
 ENV DASK_TLS_DIR=${DASK_DIR}/tls
 
-RUN mkdir -p ${DASK_LIB_DIR} ${DASK_TLS_DIR}
+USER root
+
+RUN mkdir -p ${DASK_LIB_DIR} ${DASK_TLS_DIR} && \
+    fix-permissions ${DASK_LIB_DIR} && \
+    fix-permissions ${DASK_TLS_DIR}
```

Comment:
What permissions are set?

Reply:
Read, write and execution permissions for everyone in the group to which the user of the container belongs to.

Reply:
Why do we need to set those permissions? DASK_LIB_DIR contains libraries, it's enough for the user to be able to read, which should be the case with the default permissions. The same goes for DASK_TLS_DIR, which contains certificates and keys.

Reply:
The user cannot access those directories, as they are created by root. Do you want me to replace those instructions with `chmod` and to readonly?